app/dashboard/templates/shared/rain.html

```diff
@@ -0,0 +1,32 @@
+{% comment %}
```

Comment:
theres already a rain.html template.. can we just reuse that one?

Reply:
@eswarasai [here](https://github.com/gitcoinco/web/blob/master/app/retail/templates/shared/rain.html) it is ^_^